src/Umbraco.Web/Models/Mapping/EntityMapDefinition.cs

```diff
@@ -177,6 +177,15 @@ private static void Map(ISearchResult source, SearchResultEntity target, MapperC
 
             target.Name = source.Values.ContainsKey("nodeName") ? source.Values["nodeName"] : "[no name]";
 
+            if (source.Values.ContainsKey(UmbracoExamineIndex.UmbracoFileFieldName))
```

Comment:
This would be better and more efficient if changed to `if (source.Values.TryGetValue(UmbracoExamineIndex.UmbracoFileFieldName, out var umbracoFile))...`

Reply:
Done